docs/pages/upgrading/cloud-cluster-updates.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Cloud Cluster Updates
+description: Provides a high-level overview of Teleport cluster updates on Cloud.
+---
+
+Teleport Enterprise (Cloud) clusters composed of Auth and Proxy Services are automatically
+kept up to date with patches and minor releases following the release schedule
+described in [Teleport Upcoming Releases](../upcoming-releases.mdx).
+
+- Major version updates for the cluster typically occur one month after the
+release of a new major version. Minor version updates and patches occur more regularly.
+- Major version updates will not occur if any connected Teleport Agents are more than one major version behind.
+- Updates only occur during your scheduled maintenance window.
+- Teleport Agents are only updated automatically if you enroll them in [managed updates](agent-managed-updates.mdx).
+
+## Notifications
+
+Teleport Enterprise (Cloud) users will receive notification in the Web UI when
+a cluster update is scheduled. The notification will include the version
+and maintenance window during which the update will take place.
+
+![notification-scheduled](../../img/cloud/notification-scheduled.png)
+![notification-message](../../img/cloud/notification-message.png)
+
+After the update has been completed, users will receive a new notification
+indicating that the update has finished.
+
+![notification-complete](../../img/cloud/notification-complete.png)
```

Comment:
I don't know if we really need screenshots of the notification in the docs. 

We don't really lose much information by removing them, and it's one less thing to keep up to date as the UI changes.

Reply:
I thought about including them short term since customers may not know what to look for and removing them after a few months.